frontend/stylesheets/components/_list-sponsor.scss

```diff
@@ -45,6 +68,7 @@
   }
 
   .list-sponsor__link { 
+    text-align: center;
```

Comment:
This can be removed if the sponsor images are all the same size. Also, prefer to leave `@include` as the first style and move other non-sass styles below:

```scss
.list-sponsor__link { 
  @include make-col(12);

  text-align: center;
```